src/parser/csql_grammar.y

```diff
@@ -26290,6 +26290,13 @@ pt_check_identifier (PARSER_CONTEXT *parser, PT_NODE *p, const char *str,
   char *invalid_pos = NULL;
   int composed_size;
 
+  if (strchr (str, '[') || strchr (str, ']'))
+    {
+      PT_ERRORf (this_parser, p,
+		 "Identifier name [%s] not allowed. It cannot contain '[' or ']'.",
+		 str);
```

Comment:
```suggestion
      PT_ERRORf (this_parser, p,
		 "Identifier name [%s] not allowed. It cannot contain '[' or ']'.",
		 str);
      return NULL;
```

Reply:
Returning NULL causes ane->info.name.original to be NULL, and the core is raised by executing intl_identifier_casecmp(NULL, "...") on path_expression.

I know this in itself is a problem.
However, the purpose here is to pass the existing test cases as they are.

Problems caused by returning NULL will be resolved through other issues.